target_csv.py

```diff
@@ -38,13 +38,13 @@ def flatten(d, parent_key='', sep='__'):
     return dict(items)
 
 
-def persist_messages(delimiter, quotechar, messages, destination_path, fixed_headers):
+def persist_messages(delimiter, quotechar, messages, destination_path, fixed_headers, filename_include_date):
     state = None
     schemas = {}
+    streamname_to_filename = {}
```

Comment:
This isn't being populated. Looks like I pushed up the wrong commit here. I'll close this PR.